c2corg_ui/static/partials/suggestions/articles.html

```diff
@@ -0,0 +1,6 @@
+  <span class="suggestion-text"> 
+    <span class="suggestion-title">
+      <span ng-bind-html="highlight(label, _query)"></span> 
+    </span><br>
+  </span>
+  <span class="icon-container"><span ng-bind-html="::activitiesHtml"></span></span>
```

Comment:
Does this add the activity icons to the right of the article suggestion?
That sounds relevant indeed but won't we have the same problem than with the article detail view titles when there are a lot of activities?
We can try as is but I'm curious if this will be OK.